choco/tools/chocolateyinstall.ps1

```diff
@@ -9,14 +9,22 @@ $packageArgs = @{
   checksumType  = 'md5'
 }
 
-Install-ChocolateyZipPackage @packageArgs
+# get Package Parameters
+$pp = Get-PackageParameters
 
+Install-ChocolateyZipPackage @packageArgs
 
 Write-Host "Running tlmgr path add"
 # Adds to Path
 $statementsToRun = "/C `"$toolsDir\TinyTeX\bin\win32\tlmgr.bat path add`""
 Start-ChocolateyProcessAsAdmin $statementsToRun "cmd.exe"
 
+# checks if choco switch var is in parameters
+if ($pp['SystemVar'] -eq 'true') {
+	Write-Host "Running tlmgr SystemVar path add"
+	# Adds to SystemVar Path
+	[Environment]::SetEnvironmentVariable("PATH", $Env:PATH + ";C:\Program Files\Scripts", [EnvironmentVariableTarget]::Machine)
+}
```

Comment:
If I understand the code correctly it adds `C:\Program Files\Scripts` to the system path. Is there any reason why `C:\Program Files\Scripts` needs to be added to the system path(TinyTex is installed to `C:\tools\TinyTeX` usually, where `C:\tools` can be customised)?

By the way, the installation can be added to the system path using the command `tlmgr path --w32mode=admin add` (I think this should be the default).

On the other hand, `SystemVar` isn't clear can you rename it to something else like `AddToSystemPath` instead?

Reply:
Thanks for your feedback, I accidentally copied the wrong code from my local files. Fixed it.

Does this mean that the installation should be touched again to add it to the system path?

A good point, I just renamed it to match your suggestion. Thanks